docs/reference/cli.md

```diff
@@ -406,7 +406,7 @@ uv init [OPTIONS] [PATH]
 
 <p>For example, spinners or progress bars.</p>
 
-</dd><dt><code>--no-python-downloads</code></dt><dd><p>Disable automatic downloads of Python</p>
+</dd><dt><code>--no-python-downloads</code></dt><dd><p>Disable automatic downloads of Python. [env: &quot;UV<em>PYTHON</em>DOWNLOADS=never&quot;]</p>
```

Comment:
I guess this doesn't show for any other options.

Reply:
Eek, yeah. Ugh. Will figure out a solution...